pkg/controller/node/node_controller.go

```diff
@@ -1062,11 +1064,10 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 			return ctrl.syncStatusOnly(pool)
 		}
 
-		klog.V(4).Infof("Continuing updates for layered pool %s", pool.Name)
-	} else {
-		klog.V(4).Infof("Pool %s is not layered", pool.Name)
+		klog.Infof("Continuing updates for layered pool %s", pool.Name)
 	}
 
+	klog.Info("getting nodes for pool")
```

Comment:
Maybe add the pool name that we are getting the nodes for.

Same for the other info logs below, but maybe those were for your own debugs and have been left behind